controls/srg_gpos/SRG-OS-000073-GPOS-00041.yml

```diff
@@ -11,4 +11,6 @@ controls:
             - set_password_hashing_algorithm_systemauth
             - set_password_hashing_min_rounds_logindefs
             - accounts_password_all_shadowed_sha512
+            - var_password_hashing_algorithm_pam=sha512
```

Comment:
It would be good to already set `yescrypt` and its respective number of rounds.

Reply:
I don't now if `yescrypt` is a FIPS approved cryptographic function, propose to keep for now.